src/components/Filter.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { startedSearching, tickerSearched } from '../redux/tickers/actions';
+
+const Filter = () => {
+  const dispatch = useDispatch();
+  const handleChange = (e) => {
+    dispatch(startedSearching());
+    dispatch(tickerSearched(e.target.value));
+  };
```

Comment:
- [ ] (Optional) It would be nice to create a state that handles the input whenever you are searching for an item. Currently, when I type on the search field and clear the text field, the page remains empty which means the state is not changing. I would advise you to use `useState` to listen for changes in the text field. 👍 

This video below explains what I am talking about

![chrome-capture (1)](https://user-images.githubusercontent.com/77106962/163596260-4f51b070-6a0e-4e28-af7c-bf7f84e85b7b.gif)

Reply:
Thanks, I will definitely work on this.